File: gestao-atendimento-web/src/app/pages/cliente/components/cliente-form/model/cliente-request-dto.ts
export interface ClienteDto {
  id?: string;
  tipoUsuario: number;
  nome: string;
  cpf: string;
  whatsapp?: string;
  email: string;
  senha: string;
  enderecoComercial: EnderecoDto;
  observacao?: string;
}

interface EnderecoDto {
  rua: string;
  numero?: number;
  bairro?: string;
  cep?: string;
  cidade: string;
  estado: string;
}

File: gestaoAtendimentoApp/src/main/java/com/br/diegoformentin/gestaoatendimentoapp/dto/cliente/ClienteRequestDto.java
package com.br.diegoformentin.gestaoatendimentoapp.dto.cliente;

import com.br.diegoformentin.gestaoatendimentoapp.dto.EnderecoCadastroDto;
import lombok.Builder;
import lombok.Data;

import java.util.UUID;

@Data
@Builder
public class ClienteRequestDto {
    private UUID id;
    private Integer tipoUsuario;
    private String nome;
    private String cpf;
    private String whatsapp;
    private String observacao;
    private String email;
    private String senha;
    private EnderecoCadastroDto enderecoComercial;
}
